roles/openshift_master/tasks/main.yml

```diff
@@ -27,6 +27,9 @@
   action: "{{ ansible_pkg_mgr }} name={{ openshift.common.service_type }}-master{{ openshift_version | default('') | oo_image_tag_to_rpm_version(include_dash=True) }} state=present"
   when: not openshift.common.is_containerized | bool
 
+- name: Install bash completion for oc tools
+  action: "{{ ansible_pkg_mgr }} name=bash-completion state=present"
```

Comment:
I am uncertain. I put it into master because I only ever imagined running
the oc/oadm commands on a master node.

On Wed, Jul 13, 2016 at 1:35 PM, Jason DeTiberus <notifications@github.com>
wrote:

> In roles/openshift_master/tasks/main.yml
> <https://github.com/openshift/openshift-ansible/pull/2148#discussion_r70671770>
> :
>
> > @@ -27,6 +27,9 @@
> >    action: "{{ ansible_pkg_mgr }} name={{ openshift.common.service_type }}-master{{ openshift_version | default('') | oo_image_tag_to_rpm_version(include_dash=True) }} state=present"
> >    when: not openshift.common.is_containerized | bool
> >
> > +- name: Install bash completion for oc tools
> > +  action: "{{ ansible_pkg_mgr }} name=bash-completion state=present"
>
> Should this be pulled in with openshift_cli rather than openshift_master?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/openshift-ansible/pull/2148/files/192a5c2e770325cc95deb5071b99ed7373968e38#r70671770>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AACBYeTQUwgd_NJJFBMIf_faNmZs_Ih9ks5qVSHvgaJpZM4JLh0m>
> .
>



-- 
Tim Bielawa, ${::title}
Cell: 919.332.6411  | IRC: tbielawa (#openshift)
1BA0 4FAB 4C13 FBA0 A036  4958 AD05 E75E 0333 AE37